src/components/ha-sidebar.ts

```diff
@@ -720,14 +720,10 @@ class HaSidebar extends SubscribeMixin(LitElement) {
           background-color: var(--sidebar-background-color);
           width: 100%;
           box-sizing: border-box;
-          padding-bottom: calc(
-            14px + var(--safe-area-inset-bottom, var(--ha-space-0))
-          );
+          padding-bottom: calc(14px + var(--safe-area-inset-bottom, 0));
```

Comment:
```suggestion
          padding-bottom: calc(14px + var(--safe-area-inset-bottom, 0px));
```

This needs to be 0px